imaginate_api/utils.py

```diff
@@ -101,9 +101,9 @@ def build_image_from_url(url):
   )
 
 
-# helper function that returns a timestamp as an integer, no matter if day is inputed as a timestamp or day #
-# WARNING, do not enter a timestamp from BEFORE august 1st 2024.
-def calculate_date(day: str | int):
+# Helper function that returns a timestamp as an integer, where day is a timestamp or day number
+# WARNING: Do not enter a timestamp BEFORE August 1st 2024
```

Comment:
Should we just add a check here to raise an error if date is early?

Reply:
Upon investigation, I think it's fine to leave it as is. Would be non-trivial to verify if the early date was meant to be inputted as integer (1, 1000, 100000 etc.) or as an actual UNIX timestamp if that makes sense.